DemoServer/Controllers/Demos/CompareExchange/CreateCompareExchange/CreateCompareExchangeController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using DemoServer.Utils.Cache;
+using DemoServer.Utils.Database;
+using DemoServer.Utils.UserId;
+using Microsoft.AspNetCore.Mvc;
+#region Usings
+using Raven.Client.Documents.Operations.CompareExchange;
+#endregion
+
+namespace DemoServer.Controllers.Demos.CompareExchange.CreateCompareExchange
+{
+    public class CreateCompareExchangeController : DemoCodeController
+    {
+        public CreateCompareExchangeController(UserIdContainer userId, UserStoreCache userStoreCache, MediaStoreCache mediaStoreCache,
+            DatabaseSetup databaseSetup) : base(userId, userStoreCache, mediaStoreCache, databaseSetup)
+        {
+        }
+
+        [HttpPost]
+        public IActionResult Run(RunParams runParams)
+        {
+            string cmpXchgKey = runParams.CmpXchgKey ?? "abc@gmail.com";
+            string cmpXchgValue = runParams.CmpXchgValue?? "employee/1-A";
+            
+            string result = null;
+            
+            #region Demo
+            #region Step_1
+            var putCmpXchgOperation =
+                new PutCompareExchangeValueOperation<string>(cmpXchgKey, cmpXchgValue, 0);
+            
+            CompareExchangeResult<string> putCmpXchgResult =
+                DocumentStoreHolder.Store.Operations.Send(putCmpXchgOperation);
+            #endregion
+            
+            #region Step_2
+            var success = putCmpXchgResult.Successful;
+            var putValue = putCmpXchgResult.Value;
+            var putVersion = putCmpXchgResult.Index;
+            
+            if (success == false)
+                result = "Key already exists";
+            #endregion
+            
+            #region Step_3
+            var getCmpXchgOperation =
+                new GetCompareExchangeValueOperation<string>(cmpXchgKey);
+            
+            CompareExchangeValue<string> getCmpXchgResult =
+                DocumentStoreHolder.Store.Operations.Send(getCmpXchgOperation);
+            #endregion
+            
+            #region Step_4
+            var key = getCmpXchgResult.Key;
+            var value = getCmpXchgResult.Value;
```

Comment:
For some reason, syntax coloring is reacting strange to "value"

![image](https://user-images.githubusercontent.com/507133/148260042-ece64206-e2de-424b-974b-9061bd9eaf33.png)

Reply:
The color scheme of this editor decides the coloring
So for now I added the prefix '`current`', 
which is not so bad, it reflects that this is the current data we just got...


![image](https://user-images.githubusercontent.com/17376974/148270146-453cd952-b7eb-476f-b77d-6bf3349f7e67.png)